express/code/styles/styles.css

```
@@ -209,6 +209,11 @@ main svg.icon-milo, main img.icon-milo {
 }
 
 
+.text-block .foreground > .cta-container .body-m.action-area a {
+  margin-top: unset;
```

To align the spacing better from what's coming from Milo, I would suggest:

```suggestion
  margin: unset;
```

This will remove the extra spacing being added between the buttons too. I'll defer to you though for the Express related design specs.